pkg/api/v1alpha1/cluster_types.go

```diff
@@ -373,6 +373,19 @@ type ClusterStatus struct {
 	// Descriptive message about a fatal problem while reconciling a cluster
 	// +optional
 	FailureMessage *string `json:"failureMessage,omitempty"`
+	// EksdReleaseRef defines the properties of the EKS-D object on the cluster
+	EksdReleaseRef EksdReleaseRef `json:"eksdReleaseRef,omitempty"`
+}
+
+type EksdReleaseRef struct {
+	// ApiGroup refers to the EKS-D API group
+	ApiGroup string `json:"apiGroup"`
```

Comment:
Should we call this ApiVersion to be more in line with the cluster-api templates?